### PR TITLE
Small edits to UserGuide and DevGuide

### PR DESCRIPTION
**Pull request by iamjackslayer**
Edited Class Diagram image in the user guide. Added three user stories in dev guide.

### CODE REVIEW

**Comment by jjlee050** [comment]
How come all of us are assign to this?

**Comment by iamjackslayer** [comment]
I just wanna make sure the class diagram accurately shows dependencies between components each of us is working on.

**Comment by iamjackslayer** [comment]
Any missing outs in the class diagrams I will fill them up in a future.

**Comment by iamjackslayer** [comment]
Oh wait apologize I i should not assign as assignees, should be reviewers.

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -849,7 +849,13 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |medical clinic receptionist |delete a patient record |ensure records of only active patients are kept
 
-|`* * *` |medical clinic receptionist |assign a patient to a queue for any room |patients can begin consultation quickly
+|`* * *` |medical clinic receptionist |assign a patient to a queue for any room |consultation can take place systematically and fairly
+
+|`* * *` |medical clinic receptionist |assign a queue number for each patient |patient can be called for consultatoin using his/her queue number
```
Comment (gingivitiss) [inline docs/DeveloperGuide.adoc]: spelling error for consultation

**Comment by iamjackslayer** [comment]
@jjlee050 
"The arrow is not correct as AddressBook implements ReadOnlyAddressBook which extend to VersionedAddressBook"

VersionedAddressBook is the one extending ReadOnlyAddressBook and not the other way round. Arrows seem correct. Am I missing something regarding this part?

**Comment by jjlee050** [comment]
The arrow for VersionedClinicIORecord to ClinicIORecord is solid line. I gave the wrong image. my apologies.

**Comment by iamjackslayer** [comment]
Ok. Btw fields are optional for class diagram (they can clutter the diagram). I only include Patient's fields for the time being.

**Comment by jjlee050** [comment]
Actually, yes. You can ignore first for now. Coz if we were to add all fields for all class (Model class). That is not ideal and efficient.

**Comment by arsalanc-v2** [comment]
Can the red lines be removed?

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -849,7 +849,13 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+
+|`* * *` |medical clinic receptionist |assign a patient to a queue based on his/her nric|i do not have to scroll through the displayed patient list to look for the patient's index
+
+|`* * *` |medical clinic receptionist |view the length of the queue|respond to quickly and accurately to patients asking for the number of patients in the queue
```
[the remark this answers is not in the document]
Comment (arsalanc-v2) [reply docs/DeveloperGuide.adoc]: The "to" which occurs after "respond" can be removed

**Pull request merged**